.changeset/rude-bananas-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-liveness": major
```

Comment:
Does that mean we plan to major bump the package next time?

Reply:
yep!